code-examples/find-duplicate-number/src/FindDuplicateTest.java

```diff
@@ -0,0 +1,50 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FindDuplicateTest {
+    private FindDuplicate fd = new FindDuplicate();
+
+    @Test
+    public void findDuplicate() {
```

Comment:
I only included this test (and the first method `findDuplicateSingle()`) because the question had two parts.